stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -149,6 +138,104 @@ private static void populateClientConnectorConfig(MapValue<String, Object> clien
         return customHeaders;
     }
 
+    /**
+     * Populate the retry config.
+     *
+     * @param retryConfig a retry config
+     * @param retryConnectorConfig retry connector config
+     */
+    private static void populateRetryConnectorConfig(MapValue<String, Object> retryConfig,
+                                                     RetryContext retryConnectorConfig) {
+        setInterval(Integer.parseInt(retryConfig.get(INTERVAL).toString()), retryConnectorConfig);
+        setBackOfFactor(Float.parseFloat(retryConfig.get(BACK_OF_FACTOR).toString()),
+                retryConnectorConfig);
+        setMaxInterval(Integer.parseInt(retryConfig.get(MAX_INTERVAL).toString()),
+                retryConnectorConfig);
+        setMaxAttempts(Integer.parseInt(retryConfig.get(MAX_COUNT).toString()),
+                retryConnectorConfig);
+    }
+
+    /**
+     * Validate and create the webSocket service.
+     *
+     * @param clientEndpointConfig a client endpoint config
+     * @param strand a strand
```

Comment:
```suggestion
     * @param strand a strand
```
Can we explain this a bit more in the description?